grid_map_loader/src/GridMapLoader.cpp

```diff
@@ -64,5 +64,6 @@ void GridMapLoader::publish()
   grid_map_msgs::msg::GridMap message;
   grid_map::GridMapRosConverter::toMessage(map_, message);
   publisher_->publish(message);
+  rclcpp::sleep_for(std::chrono::nanoseconds(duration_.nanoseconds()));
```

Comment:
or just duration_.sleep() ...

Reply:
Not sure if this method is part of the [rclcpp::duration](https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/duration.hpp) class.

Reply:
That was unexpected, OK, makes sense then